src/setOfPeople.js

```diff
@@ -38,7 +38,14 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const setPeople = new Set();
+
+  people.forEach((item) => !setPeople.has(item)
```

Comment:
краще йти від `true`, а не `false`
```suggestion
  people.forEach((item) => setPeople.has(item)
```